index.html

```diff
@@ -11,7 +11,7 @@
     <title></title>
   </head>
   <body class="h-full bg-pureWhite font-roboto">
-    <div id="root" class="flex flex-col gap-y-7 items-center py-5 px-2.5"></div>
+    <div id="root" class="flex flex-col gap-y-7 items-center"></div>
```

Comment:
Be careful using index to style your error page. It's better to apply these styles directly to the component that it's intended for as this will affect everything.

Reply:
If this was meant to be a global adjustment then this is fine.

Reply:
yes, this is a global adjustment because these paddings create the white frame.